dashboards-observability/public/components/trace_analytics/components/dashboard/mode_picker.tsx

```diff
@@ -0,0 +1,97 @@
+import { EuiButtonEmpty, EuiPopover, EuiPopoverTitle, EuiSelectable } from "@elastic/eui";
```

Comment:
missing license header

nit. could you also run prettier, the indentation is inconsistent with other files